api/src/main/proto/stellarstation/api/v1/monitoring/monitoring.proto

```diff
@@ -0,0 +1,235 @@
+  radio.LineCoding line_coding = 4;
+
+  // The bandwidth of the filter.
+  uint64 filter_bandwidth_hz = 5;
+
+  // The sampling frequency.
+  uint64 sampling_frequency_hz = 6;
+}
+
+// The current configuration of a receiver. A receiver is taking a waveform from a satellite and
+// applying signal processing to produce a bitstream.
+message ReceiverConfiguration {
+  // The modulation type of the receiver.
+  radio.Modulation modulation = 1;
+
+  // The bandwidth, in Hz.
+  uint64 bandwidth_hz = 2;
+
+  // The sampling frequency.
+  uint64 sampling_frequency_hz = 3;
+
+  // The bitrate of the receiver.
+  uint64 bitrate = 4;
+
+  // The line coding of the receiver.
+  radio.LineCoding line_coding = 5;
+
+  // The frame synchronization word being used. Unset if frame synchronization is disabled.
+  common.Bits synchronization_word = 6;
+
+  // Whether convolutional code descrambling is enabled.
+  bool is_convolutional_code_descrambling = 7;
+
+  // Whether convolutional code decoding is enabled.
+  bool is_convolutional_code_decoding = 8;
+
+  // Whether convolutional codes are inverted.
+  bool is_convolutional_code_inverted = 9;
+
+  // Whether reed solomon coding is being used.
+  bool is_reed_solomon_enabled = 10;
+
+  // The size of a frame, in bits.
+  uint32 frame_size = 11;
+
+  // Whether frame checking is enabled.
+  bool is_frame_checking_enabled = 12;
+}
+
+// The current configuration of a ground station. This is controlled by the parameters of a pass,
+// based on the configuration of the satellite it will communicate with. When debugging issues with
+// a pass, it is good to first confirm that the actual reported configuration matches the expected
+// settings.
+message GroundStationConfiguration {
+
+  // The current configuration of the transmitter in use by the ground station. This configuration
+  // should match the parameters of the executing pass. When debugging issues with data
+  // transmission (e.g., no response from satellite), it can be useful to confirm these values match
+  // the expected configuration of the transmitter.
+  TransmitterConfiguration transmitter = 1;
+
+  // The current configuration of the receiver in use by the ground station. This configuration
+  // should match the parameters of the executing pass. When debugging issues with data
+  // reception (e.g., can't decode signal), it can be useful to confirm these values match
+  // the expected configuration of the receiver.
+  ReceiverConfiguration receiver = 2;
+
+  // The current configuration of the antenna used by the ground station. These are the physical
+  // parameters of an antenna and in practice should never change, even across passes. But natural
+  // events such as earthquakes can cause small changes in these values. When debugging issues with
+  // tracking a satellite (e.g., getting no signal at all), it can be useful to confirm these values
+  // match the expected configuration of the antenna.
```

Comment:
Ah so heading, roll, pitch are fixed also?

Reply:
@vaelen Can you confirm? I think they're the fixed rotation of the base of the antenna, which is basically screwed into the Earth. Azimuth / elevation are the moving angles.

Reply:
I'm actually not sure.  We need to ask the ground station vendor to clarify.

Reply:
@anuraaga pitch, roll, heading (aka yaw) is used for airplanes and satellites because they have 3 free axes of rotation. A ground station has only 2 free axes and its pointing is described with AZ/EL.

Reply:
These three angles are the estimated yaw, pitch, roll of the satellite reported by the ACU. The ACU has an automatic mode to report these parameters.

Since there is no functional description of this **"automatic attitude mode"** (it only appears listed on page 15), @vaelen will request Zodiac guys the user manual of the ACU.

Reply:
@anuraaga we can remove this parameters from the API since: (1) not all supported modems / radios will have this option, (2) we do not know if the attitude of all satellites can be estimated with this mode.